### ISSUE TITLE
Body Fit Gym

### ISSUE THREAD

**Issue by dtay2525** [dtay2525/Bodyfit#1]
I started my process by creating a title for the tab of the website “Body Fit for Life” in the <title> tag then added the business logo utilizing the <img> tag and <h1> for the main header of the web page. In the next step I added the description of the product being advertised by utilizing the <p> tag and <strong> tag to emphasize importance of the <h2> heading, making it bold. I also decided to utilize the same <strong> tag to bold each one of the terms XJS500 throughout the web page. 
	In the second section of the web page, I decided to wrap my text around an image of the product being advertised by using the align attribute. I then used three <br> tags in order to create more white space between elements and one <hr>horizontal rule to create an established end of section. In addition, I created an <h2> heading and added a style attribute to increase the font-size and look. Listed under the features section I created a description list using <dl> tag and <dt>tag to list each feature and its description, each feature is listed with the <strong> tag and style attribute, font-size 140%. 
	In the final section of the Body fit for life web page I added a final section called Dimensions which again was listed within the <h2> header however this time in order to list the dimensions I created an <ul> unordered list with the <li> tag for each dimension and I used the <address> tag to add in my author information including the necessary information requested on the project which included my name, address, student identification, lesson number and email address. The final decision I made was creating a style1.css web page to create the styles and formatting needed to present a great looking web page.
	In conclusion, my previous experience with c sharp programming really helped me find my footing in the world of html. Creating the structure and finding the vision for a positive experience for the end user while mapping a readable code that flows from top down.
